Makefile

```diff
@@ -1,3 +1,8 @@
+.PHONY: docs
+docs: cli
+	rm -fr docs/docs/references/cli/rill*
+	./rill docs generate docs/docs/references/cli/
```

Comment:
1. Target should be `docs.generate`
2. Use `go run ./cli docs generate docs/docs/references/cli/` instead, and remove dependency on `cli` target (to prevent full frontend build, which is slow)

Reply:
Item 2 conflicts with
> Also, move the generated docs to references/rill-cli-usage instead of references/cli/rill/ to prevent too much nesting

Reply:
Oh, I just meant to use `go run ./cli` instead of `./rill` (a prod build), not that the arg there needs to match exactly